### DISCUSSION TITLE
Training within Docker image?

### DISCUSSION THREAD

**Discussion by genert** [explosion/spaCy#8616] [Help: Coding & Implementations] [answered]
Are there any working Dockerfiles or project examples where training occurs within Docker image?

Asking because I want to create `python -m spacy project run train` entry within Docker container, and run it as AWS Sagemaker Training Job.

However the latter requires Docker image for it to work.

**Comment by polm** [comment, marked as answer]
We don't have any example projects using Docker specifically, though I think if you search around you can probably find some community examples.

spaCy should work without any special considerations in Docker.
